sources/packages/backend/libs/services/src/system-users/system-users.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { User } from "@sims/sims-db";
+import { Repository } from "typeorm";
+import { SERVICE_ACCOUNT_DEFAULT_USER_EMAIL } from "../../../utilities/src/system-configurations-constants";
```

Comment:
Please review the `import` to use the shortcut `@sims/utilities`.

Reply:
oh..this automatically happens when we rename a folder

Reply:
Yes, I got it. When I was moving the files lately I was avoiding the automatic import fix to force the compile errors and adjust them manually.